<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Balance</class>
 <widget class="QWidget" name="Balance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>255</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border:none;
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="containerBalance" native="true">
     <property name="minimumSize">
      <size>
       <width>252</width>
       <height>640</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#containerBalance{
border: none;
background-color: qlineargradient(y1:0, y2: 1, stop: 0 #ffffff, stop: 1 #e7eefd);
padding-left:10px;
padding-right:10px;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="vLayoutBalance" stretch="0,6">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QPushButton" name="pushButton">
          <property name="minimumSize">
           <size>
            <width>213</width>
            <height>150</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">background-colot:transparent;
border:none;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/icons/simbol-png</normaloff>:/icons/simbol-png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>106</width>
            <height>92</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,4">
          <item alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="btnBalance">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color:transparent;</string>
            </property>
            <property name="text">
             <string>Total Balance</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/icons/ic-information-png</normaloff>:/icons/ic-information-png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="labelBalance">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">#labelBalance{
background-color: transparent;
color: #707070;
font-size: 24px;
}</string>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:medium; font-weight:600;&quot;&gt;100000.00 Veil&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>1</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:#bababa;
margin-left:15px;
margin-right:15px;</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="styleSheet">
               <string notr="true">background-color: transparent;
color: #707070;</string>
              </property>
              <property name="text">
               <string>Unspendable</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelUnconfirmed">
              <property name="styleSheet">
               <string notr="true">background-color: transparent;
color: #707070;</string>
              </property>
              <property name="text">
               <string>50.738 VEIL</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>10</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="styleSheet">
               <string notr="true">background-color: transparent;
color: #707070;</string>
              </property>
              <property name="text">
               <string>Immature</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelImmature">
              <property name="styleSheet">
               <string notr="true">background-color: transparent;
color: #707070;</string>
              </property>
              <property name="text">
               <string>50.738 VEIL</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
             <widget class="QLabel" name="labelQr">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>100</height>
               </size>
              </property>
              <property name="whatsThis">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;background-color: transparent;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: transparent;
margin:0 0 20 10;</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="horiz">
              <property name="styleSheet">
               <string notr="true">#horiz{
padding-left:5px;
padding-right:5px;
}</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="labelReceive">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">   background-color: transparent;
color: #004377;</string>
                 </property>
                 <property name="text">
                  <string>Receiving address</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="copyAddress">
                 <property name="maximumSize">
                  <size>
                   <width>24</width>
                   <height>24</height>
                  </size>
                 </property>
                 <property name="focusPolicy">
                  <enum>Qt::NoFocus</enum>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">   background-color: transparent;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>:/icons/ic-copy-blue-png</normaloff>:/icons/ic-copy-blue-png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>18</width>
                   <height>18</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelAddress">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: transparent;
padding-left:5px;
padding-right:5px;
padding-bottom:3px;
font-size:10px;
color:#707070;</string>
              </property>
              <property name="midLineWidth">
               <number>0</number>
              </property>
              <property name="text">
               <string>1N6i46dytMPVhV1JMGZFuQBh7BZZ6nNLox</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <property name="indent">
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
